conda-forge.yml

```diff
@@ -1,4 +1,3 @@
-bot: {automerge: true}
```

Comment:
I'm removing the automerge bot because I think that's causing trouble and dependencies aren't getting updated when they need to.  In my view, the bot is only useful for packages that have fixed dependencies without constraints that could change with time.  `pip check` will not notice that constraints are being violated unless an out-of-range version actually gets installed during testing, which is often not the case.